src/ensembl/src/content/app/entity-viewer/shared/helpers/transcripts-sorter.ts

```diff
@@ -0,0 +1,50 @@
+import sortBy from 'lodash/sortby';
+
+import { getFeatureLength } from './entity-helpers';
+
+import { Transcript } from '../../types/transcript';
+
+function compareTranscriptLengths(
+  transcriptOne: Transcript,
+  transcriptTwo: Transcript
+) {
+  const transcriptOneLength = getFeatureLength(transcriptOne);
+  const transcriptTwoLength = getFeatureLength(transcriptTwo);
+
+  if (transcriptOneLength < transcriptTwoLength) {
+    return -1;
+  }
+
+  if (transcriptOneLength > transcriptTwoLength) {
+    return 1;
+  }
+
+  return 0;
+}
+
+export function defaultSort(transcripts: Transcript[]) {
+  const transcriptsWithCds = transcripts
+    .filter((transcript) => transcript.cds)
+    .sort(compareTranscriptLengths);
+
+  const transcriptsWithoutCds = transcripts.filter(
+    (transcript) => !transcript.cds
+  );
```

Comment:
since cds is not the same as ccds anyway, let's remove the with/without cds split from the code.